packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -27,7 +27,7 @@ export function createAuthMiddleware({
       const token = parts[1]
       if (
         process.env.NODE_ENV !== 'production' &&
-        token === config.devAccessToken
+        (token === config.devAccessToken || config.bypassSignatureValidation)
```

Comment:
I think the check should be moved below to wrap:
https://github.com/interledger/rafiki/blob/e796dedeee7c3aedd43c5c977ce83f420ea8667c/packages/backend/src/open_payments/auth/middleware.ts#L48-L55